### PR TITLE
Fixes #395 Document steps for installation to Heroku

### LINKED ISSUE #395: Document steps for Yaydoc installation to Heroku
### This is a
- [ ] Bug report
- [ ] Feature request
- [x] Documentation request

### Description
Updating documentation by documenting steps for Yaydoc installation to Heroku would help new developers deploy and test the application more effectively.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue #395 
## Motivation and Context

Updating documentation by documenting steps for Yaydoc installation to Heroku would help new developers deploy and test the application more effectively.
## Types of changes

- [ ] Bug fix 
- [ ] New feature
- [x] Documentation update

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: docs/installation/heroku.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      wget -qO- https://cli-assets.heroku.com/install-ubuntu.sh | sh
+    
+*Note:-* Details for other Operating Systems can be found at https://devcenter.heroku.com/articles/heroku-cli
+
+### 2. Login to Heroku Server (Create an account if it doesn’t exist)
+* Type in the following command in your server, enter credentials and login
+
+      heroku login
+      
+### 3. Deploy Yaydoc to Heroku
```
Comment (pri22296) [inline docs/installation/heroku.md]: Is the numbering on titles needed?

File: docs/installation/heroku.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Installing Yaydoc on Heroku
+Heroku is a cloud platform as a service (PaaS) that is used as a web application deployment model. The Yaydoc Web User Interface is a NodeJS application and Heroku makes it really easy to deploy and scale Node.js applications in the cloud.
+
+## One-click Heroku deployment [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (pri22296) [inline docs/installation/heroku.md]: Why not keep the button on a separate line?

Comment (imujjwal96) [inline docs/installation/heroku.md]: okie

File: docs/installation/heroku.md (hunk 1 above)
Comment (imujjwal96) [inline docs/installation/heroku.md]: what do you think?

Comment (pri22296) [inline docs/installation/heroku.md]: I don't think so. Would be ok though If you feel it's needed.

Comment (imujjwal96) [inline docs/installation/heroku.md]: I'll change it then.

File: docs/installation/heroku.md (hunk 2 above)
Comment (imujjwal96) [inline docs/installation/heroku.md]: Updated

File: docs/installation/heroku.md (hunk 1 above)
Comment (imujjwal96) [inline docs/installation/heroku.md]: Updated.

**Pull request merged**